README.md

```diff
@@ -3,128 +3,99 @@
 [![CircleCI](https://circleci.com/gh/tulibraries/funcake_dags.svg?style=svg)](https://circleci.com/gh/tulibraries/funcake_dags)
 ![pylint Score](https://mperlet.github.io/pybadge/badges/9.47.svg)
 
-This is the repository for Funnel Cake (PA Digital / DPLA Data QA Interface) Airflow DAGs (Directed Acyclic Graphs, e.g., data processing workflows) for data indexing to Solr + related jobs. These DAGs are expecting to be run within an Airflow installation akin to the one built by our [TUL Airflow Playbook (private repository)](https://github.com/tulibraries/ansible-playbook-airflow).
+This is the repository for Funnel Cake (PA Digital / DPLA Data QA Interface) Airflow DAGs (Directed Acyclic Graphs, e.g., data processing workflows) for data indexing to Solr and related jobs. These DAGs are expecting to be run within an Airflow installation akin to the one built by our [TUL Airflow Playbook (private repository)](https://github.com/tulibraries/ansible-playbook-airflow).
 
-## Repository Structure
 
-This repository has 3 main groups of files:
-- Airflow DAG definition python files (ending with `_dag.py`);
-- Airflow DAG tasks python files used by the above (starting with `task_`);
-- and required local development, test, deployment, and CI files (`tests`, `configs`, `.travis`, Pipfile, etc.).
-
-## Airflow Expectations
-
-These the Airflow expectations for these Funnel Cake DAGs to successfully run:
+## Prequisites
 
 **Libraries & Packages**
 
-- Python 3.7
-- Python Packages: see the [Pipfile](Pipfile)
+- Python: Version as specified in `.python-version`, currently 3.6.8.
```

Comment:
```suggestion
- Python: Version as specified in `.python-version`.
```

I still think it makes more sense to leave it at that so the README doesn't need to be updated every time the version changes.

Reply:
IT did not make it in the first time I hit the add suggestion to branch. Will try again.